tests/utils/verification/volumeproperties.go

```diff
@@ -101,14 +102,17 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 func CheckVolumeAvailability(hostName string, volumeName string) bool {
 	log.Printf("Checking volume [%s] availability from VM [%s]\n", volumeName, hostName)
```

Comment:
this should be gocheck Logf() - possibly define a misc.Logf() that uses gocheck Logf to keep all logs uniform,